src/components/time-picker/time-picker.st.css

```diff
@@ -20,7 +21,7 @@
     width: 7.5em;
 }
 .root {
-    -st-states: focus, disabled, empty;
+    -st-states: focus, disabled, empty, has-placeholder, error, rtl;
```

Comment:
We have RTL state, but no CSS for RTL... Should we add CSS for RTL?

Reply:
added